EIPS/eip-5883.md

```diff
@@ -0,0 +1,110 @@
+---
+eip: 5883
+title: Token Transfer by Social Recovery
+description: On-Chain Social Recovery
+author: Erhard Dinhobl (@mrqc), Kevin Riedl (@wsdt)
+discussions-to: https://ethereum-magicians.org/t/eip-5806-delegate-transaction/11409
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-07-19
+---
+
+## Abstract
+
+Within this EIP a mechanism of a social recovery is proposed. This implies that by the approval of many identities a token transfer from the identity of the lost private key to a knew identity where the private key is known. This approval is not a technical implementation but needs an interaction of humans. These humans are - based on the Soul Bound Token proposal - called Souls. When enough Souls give their approval (which is a Yes/No decision) and a threshold is reached this transfer from an old to a new identity is performed by a smart contract.
+
+## Motivation
+
+It is always the problem that the private key of an identity can be lost. The reasons for that are manifold. If that key is lost its not possible to recover the tokens. The holder loses his tokens forever. Also the implications to the ecosystem of the token itself is a problem. The more tokens are lost the less tokens are available and a natural growth and a planned evolving of the token is not possible.
+
+
+## Specification
+
+```solidity
+
+pragma solidity ^0.8.7;
+
+interface ISBT {
+    function calcScore(address identityAddr_) external view returns (int256);
+
+    function approveTransfer(address from_, address to_) external;
+
+    function requestTransfer(address from_, address to_) external payable;
+
+    function addNeighbour(address neighbour_) external;
+
+    function removeNeighbour(address neighbour_) external;
+}
+```
+
+**The math behind it**:
```

Comment:
Can you clarify whether it's a MUST or SHOULD or MAY to apply such math algorithm in a compliant contract?

Reply:
Done!